lib/mdx.ts

```diff
@@ -66,16 +73,27 @@ export async function prepareMDX(source: MDXSource): Promise<Post> {
     )
   }
   try {
+    const { default: readingTime } = await import('reading-time')
+    const { default: rehypeCodeTitles } = await import('rehype-code-titles')
+    const { default: remarkAutoLink } = await import('remark-autolink-headings')
+    const { default: remarkExternalLink } = await import(
+      'remark-external-links'
+    )
+    const { default: remarkGfm } = await import('remark-gfm')
+    const { default: remarkSlug } = await import('remark-slug')
+
     // mdx-bundler processes the frontMatter internally from the file. No need for gray-matter anymore.
     const { code, frontmatter } = await bundleMDX(source.file, {
+      // @ts-ignore
       xdmOptions(_input, options) {
         return {
           ...options,
           remarkPlugins: [
             ...(options.remarkPlugins ?? []),
-            gfm,
-            remarkEmoji,
-            remarkExternalLinks,
+            remarkSlug,
+            [remarkAutoLink, { behavior: 'before' }],
+            remarkGfm,
+            remarkExternalLink,
```

Comment:
- [ ] Verify this works.

Reply:
Verified it is applying the correct attributes to the links as the plugin should